generators/gen_reactnative.py

```diff
@@ -61,6 +62,8 @@
 
 INDEX_TSX_WARNING = "console.log(\"index.tsx was re-generated; make sure you include any missing code\")\n\n"
 
+NOT_EXPOSED_DEFINITIONS = ["DdSdk", "DdSdkConfiguration"]
```

Comment:
I'm not in favor of having these hardcoded names here. Seems very hacky. Instead we could add a property `exposed`  in the schema.

Reply:
it is a hack indeed, which solves a very particular problem. not sure how much it will be needed in future, but I can go for the schema field.